src/vivarium_gates_iv_iron/components/hemoglobin.py

```diff
@@ -16,6 +21,70 @@ def __init__(self):
     def name(self):
         return "hemoglobin"
 
+    def setup(self, builder: Builder):
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.columns_created = ["country", "hemoglobin_distribution_propensity", "hemoglobin_percentile"]
+        # load data
+        mean = builder.data.load(data_keys.HEMOGLOBIN.MEAN)
+        stddev = builder.data.load(data_keys.HEMOGLOBIN.STANDARD_DEVIATION)
+        # TODO replace --->>>
+        location_weights = mean.copy().drop(columns=["value"])
+        # lookup data, categorical column "country"
+        mean_dfs, stddev_dfs = [], []
+        # TODO have this in artifact and correct the location weights
+        for country in ["Bangladesh", "Pakistan", "India"]:
```

Comment:
Store this list of countries somewhere sensible as a constant?

Reply:
This is temporary code.